ros_ws/src/autonomous/evolutionary/scripts/neural_car_driver.py

```diff
@@ -98,20 +110,25 @@ def mutate(self):
         offspring.load_vector(parameters)
         return offspring
 
-    def load(self):
-        self.load_state_dict(torch.load(MODEL_FILENAME))
+    def load(self, index):
+        self.load_state_dict(torch.load(
+            MODEL_FILENAME + "_" + str(index) + MODEL_FILENAME_ENDING))
```

Comment:
You could use a format string here. If `MODEL_FILENAME = 'evolutionary-{:d}'`, you can use it like this: `MODEL_FILENAME.format(index)`.